### PR TITLE
Resolve some hardcoded syms in main.s

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: asm/main.s [hunk 1]
```diff
@@ -41,11 +41,11 @@ _02000808:
 	ldr r1, _02000924 @ =0x07000000
 	mov r2, #1024
 	bl sub_200093C
-	ldr r1, _02000928 @ = 0x02000B68
+	ldr r1, _02000928 @ = _02000B68
```
Comment (martmists) [inline asm/main.s:44]: Labels should indicate what type they are or have a name matching the ghidra decomp.

File: asm/main.s [hunk 2]
```diff
@@ -81,10 +81,10 @@ _02000918: .4byte 0x027E0000
 _0200091C: .4byte 0x00000400
 _02000920: .4byte 0x05000000
 _02000924: .4byte 0x07000000
-_02000928: .4byte 0x02000B68
+_02000928: .4byte _02000B68
 _0200092C: .4byte 0x027FFF9C
 _02000930: .4byte 0x01FF8000
-_02000934: .4byte 0x02000C55 @ NdsMain
+_02000934: .4byte NdsMain+1
```
Comment (martmists) [inline asm/main.s:87]: Can you explain the `+1` here? As far as I know, NdsMain starts at 0x02000c54, why is it offset?

File: asm/rom.s [hunk 3]
```diff
@@ -1,5 +1,8 @@
 /* rom.s TODO: Disassemble */
+.include "asm/macros.inc"
 
 .section .text
 
+	thumb_func_start NdsMain
+NdsMain:
```
Comment (martmists) [inline asm/rom.s:7]: Should be renamed to `NitroMain` to match decomp.

File: asm/main.s (hunk 2 above)
Comment (ProjectRevoTPP) [reply asm/main.s:87]: Thumb functions add 1 because of the thumb flag

File: asm/rom.s (hunk 3 above)
Comment (ProjectRevoTPP) [reply asm/rom.s:7]: We can't see your Ghidra definitions

File: asm/main.s (hunk 1 above)
Comment (ProjectRevoTPP) [reply asm/main.s:44]: We're not going to have a clue what anything is

This is why early stuff in gen 3 dism is gUnknown_(Address) until identified

Comment (martmists) [reply asm/main.s:44]: At least that would show it's not e.g. a function or something like that.

File: asm/rom.s (hunk 3 above)
Comment (Cleverking2003) [reply asm/rom.s:7]: This name is from NitroSDK

File: asm/main.s (hunk 1 above)
Comment (Cleverking2003) [reply asm/main.s]: I don't think that's a problem for now

**Pull request merged**